src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let res = [];
+  const newState = { ...state };
+
+  res[0] = newState;
```

Comment:
You can remove this, according to examples in task you don't have to push original state to array:
![image](https://user-images.githubusercontent.com/118751034/215078687-4f0a8715-8048-47a4-b931-4549e5210c3e.png)